azurerm/internal/services/cognitive/resource_arm_cognitive_account.go

```diff
@@ -169,6 +167,11 @@ func resourceArmCognitiveAccountCreate(d *schema.ResourceData, meta interface{})
 		return fmt.Errorf("the QnAMaker runtime endpoint `qna_runtime_endpoint` is required when kind is set to `QnAMaker`")
 	}
 
+	qnaRuntimeEndpoint, exists := d.GetOk("qna_runtime_endpoint")
+	if exists {
+		props.Properties.APIProperties.QnaRuntimeEndpoint = utils.String(qnaRuntimeEndpoint.(string))
+	}
```

Comment:
could we add [an acceptance test](https://github.com/terraform-providers/terraform-provider-azurerm/blob/304c19a6c07659d20082c0296121accff89471c8/azurerm/internal/services/cognitive/tests/resource_arm_cognitive_account_test.go#L17-L38) to confirm this works when the `qna_runtime_endpoint` field is unset?

Reply:
OK, I will try. This is my first contribution. Is there any existing test code similar to this change I can refer to?